### PR TITLE
Intellij cheat sheet : add alias "Android Studio"

### PR DESCRIPTION
**Pull request by abadojack**
###### What does your Pull Request do (check all that apply)?

- [*] Improvement
    - [*] Enhancement: **`intellij cheat sheet: add alias "androidaStudio"`**

###### Description of changes

Add android studio as an alias to intellij.

###### People to notify (@jamie23)

---

Instant Answer Page: https://duck.co/ia/view/intellij_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @jamie23

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/intellij.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "sourceName": "Shortcut World",
         "sourceUrl": "http://www.shortcutworld.com/en/win/IntelliJ.html"
     },
-    "aliases": ["intellij idea", "intellij editor", "intellij ide", "idea ide"],
+    "aliases": ["intellij idea", "intellij editor", "intellij ide", "idea ide", "android studio"],
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/intellij.json]: @abadojack It looks like Android Studio /= Intellij IDEA - may I ask the reason for this addition?

**Comment by abadojack** [comment]
@GuiltyDolphin Android Studio is built on top of Intellij and all Intellij IDEA shortcuts apply for android studio.

**Comment by GuiltyDolphin** [comment]
@abadojack @jamie23 the triggering issue appears to be due to another cheat sheet that was in development (https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2136) - it doesn't look like it's been active in a while though.

**Comment by edgesince84** [comment]
If the triggering issue is due to the old Android Studio Cheat Sheet from @anshadshaji, hopefully we can get that sorted out quickly, since that one seems to be dead in the water.  @moollaza, thoughts?

**Comment by GuiltyDolphin** [comment]
@edgesince84 If it deploys fine then :+1: - we can always remove the alias later if someone wants to develop a full cheat sheet for it.

**Comment by GuiltyDolphin** [comment]
@jamie23 - happy with the changes? :whale2:

**Comment by jamie23** [comment]
@GuiltyDolphin The previous 'blank' answer is still triggering for me on the triggers 'android studio reference' and 'android studio cheatsheet'. Does the deletion of the previous answer take time to propagate or is there an issue on my side?

**Comment by GuiltyDolphin** [comment]
@jamie23 Oh, huh... I thought it was fixed.

@jdorweiler any ideas why removing the IA hasn't fixed it?

**Comment by zekiel** [comment]
I've moved some of the labels around to prompt a re-deploy to beta. I've also gotten rid of the broken Android Studio IA page in case it's interfering. If it's still busted after the redeploy, we'll investigate a bit more.

**Comment by GuiltyDolphin** [comment]
@zekiel label deploying is off according to @jdorweiler - I think you'll have to do it manually.

**Comment by moollaza** [comment]
Looks like we're not going to be able to remove Android Studio Cheat Sheet from Beta right now.

This LGTM :+1:

**Pull request merged**